### ISSUE TITLE
finetune adapointr on PCN vs costume dataset

### ISSUE THREAD

**Issue by m-kafiyan** [yuxumin/PoinTr#181]
Hi,
First of all, thanks for the great work and for sharing this project.

I'm fine-tuning AdaPoinTr (trained on the PCN dataset) on my own custom dataset. Before opening this issue, I carefully read through most of the existing ones but couldn't find a satisfying answer to my concern.

Here’s what I’m observing:
- During fine-tuning, the scale of the loss looks different.  For example, in dense the loss starts around 28, while in another it starts closer to 4 (considering the loss is multiplied by 1000). Is such variation expected or natural?
-  Also, I noticed a gap between the dense training loss and the dense test loss. Surprisingly, the test loss is lower than the train loss. I saw a few issues discussing similar results, but I’m still not convinced why this would happen.

Could you please clarify:

1. Is this behavior normal or a red flag?
2. If this is expected, what’s the reasoning behind it?
3. How can I reduce the dense test error further? Would adding weighting terms to the loss help?

As a sanity check, I also fine-tuned AdaPoinTr on the original PCN dataset, and I observed similar results there too.

Any insights or suggestions would be greatly appreciated!

Here is the results for my costume dataset

<img width="1535" height="490" alt="Image" src="https://github.com/user-attachments/assets/47a1bb86-c270-44ab-9d18-aaad7b658c48" />

and this is the results of fine-tuning over pcn datasset

<img width="1536" height="500" alt="Image" src="https://github.com/user-attachments/assets/31f8c5e1-5b29-4323-86ea-5b2dcd045b62" />
